mail_send/mailSend.js

```diff
@@ -0,0 +1,35 @@
+const nodemailer = require('nodemailer');
+var fs = require("fs");
+var handlebars = require('handlebars');
+module.exports = (obj) => {
+    let transporter = nodemailer.createTransport({
+        service: 'gmail',
+        auth: {
+            user: process.env.MAIL,
+            pass: process.env.PASSWORD
+        }
+    });
+    fs.readFile(__dirname + '/varification.html', (err, html) => {
+        if (err) {
+            console.log(err);
+        } else {
+            var template = handlebars.compile(html.toString());
+            url = "http://localhost:3000/user/verified/" + obj._id;
```

Comment:
what is this? localhost. this should not be hardcodedo r how will it work on server